blog-service/2022-12-02-slo.md

```diff
@@ -0,0 +1,18 @@
+---
+title: AWS ELB SLO Alert Installation via Terraform
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - sumo logic
+  - slo
+  - reliability management
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+AWS ELB users can now install Sumo Logic out-of-the-box SLOs and associated monitors using [our new Terraform script](https://github.com/SumoLogic/sumologic-solution-templates/tree/master/slo_packages/aws).
+
+This script allows you to install Sumo Logic SLOs in your specified AWS ELB directory, then configure SLO alert notifications (e.g., latency limit breached, server error limit breached). Once installed, you can view and edit your SLO alerts via Terraform or from your Sumo Logic **Monitors** page.
```

Comment:
I'd remove "**, ** **then**" and rephrase this as:
This script allows you to install Sumo Logic SLOs in your specified AWS ELB directory and configure SLO alert notifications (e.g., latency limit breached, server error limit breached. Once installed, you can view and edit your SLO alerts via Terraform or from your Sumo Logic Monitors page.

Reply:
thank you!